project/Dependencies.scala

```diff
@@ -23,29 +23,29 @@ object Dependencies {
 
   val scalaJava8Compat = Seq("org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0")
 
-  val playJsonVersion = "2.7.2"
+  val playJsonVersion = "2.7.4"
   val playJson = Seq("com.typesafe.play" %% "play-json" % playJsonVersion)
 
   val slf4jApi = Seq("org.slf4j" % "slf4j-api" % "1.7.26")
 
   val javaxInject = Seq("javax.inject" % "javax.inject" % "1")
 
-  val sslConfigVersion = "0.3.7"
+  val sslConfigVersion = "0.4.0"
   val sslConfigCore = Seq("com.typesafe" %% "ssl-config-core" % sslConfigVersion)
 
-  val scalaXmlVersion = "1.1.1"
+  val scalaXmlVersion = "1.2.0"
```

Comment:
Is `scala-xml` 1.2.0 binary compatible with `1.1.1`? If not, then the best option is to add distinct versions depending on the Scala version used.

Reply:
https://github.com/scala/scala-xml/releases/tag/v1.2.0

> This release is not backwards compatible with the last release of scala-xml, 1.1.1, because of the small change that removes the name field from transform.RewriteRule, see [#129](https://github.com/scala/scala-xml/issues/129). It appears to have been an unused vestigial class member in a rarely used class, so it's likely of low risk.